examples/DataWizard/Polygon Regions DataWizard.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
I checked this file as a notebook, and it does not have the results for each cell. do you think we should add the results so that people can see them? I think we do that on other example notebooks.